bin/offline.cpp

```diff
@@ -163,29 +163,28 @@ int main(int argc, char *argv[]) {
 
 
     util::RunLoop loop;
-    std::shared_ptr<DatabaseFileSource> fileSource = std::static_pointer_cast<DatabaseFileSource>(
-        std::shared_ptr<FileSource>(FileSourceManager::get()->getFileSource(
-            FileSourceType::Database,
-            ResourceOptions().withAccessToken(token).withBaseURL(apiBaseURL).withCachePath(output))));
+    std::shared_ptr<JointDatabaseStorage> databaseStorage = FileSourceManager::get()->getDatabaseStorage(
```

Comment:
Maybe a better name... SQLiteDatabaseStorage?

Reply:
Not so sure about adding SQLite to the API name. Are we moving to a non-SQLite implementation and doing this in order to maintain backwards compatibility?

What's problematic about the existing name?